arangod/VocBase/Validators.cpp

```diff
@@ -48,10 +48,28 @@ std::string const&  to_string(ValidationLevel level) {
   return StaticStrings::ValidatorLevelStrict;  // <- avoids: reaching end of non-void function ....
 }
 
+std::string const&  to_string(validation::SpecialProperties special) {
+  switch (special) {
+    case validation::SpecialProperties::All:
+      return StaticStrings::ValidatorPropertyAll;
+    case validation::SpecialProperties::None:
+      return StaticStrings::ValidatorPropertyNone;
+    case validation::SpecialProperties::Key:
+    case validation::SpecialProperties::Id:
+    case validation::SpecialProperties::Rev:
+    case validation::SpecialProperties::From:
+    case validation::SpecialProperties::To:
```

Comment:
is it intentional to make all these values produce a result of `ValidatorPropertyNone`?

Reply:
Yes - For now it distinguished only between `none` and `all`. The rest defaults to `none`.

Reply:
Ok